.github/ISSUE_TEMPLATE/feature.md

```diff
@@ -0,0 +1,17 @@
+---
+
```

Comment:
Don't think we need an empty line here?

Reply:
Now a template is equal to the testrunner-toolkit. I can add an empty line if it's necessary.

Reply:
I'm saying it's already an empty line ;) And I don't think we need it.

Reply:
Sorry, didn't get you properly. There is no empty line anymore ;)